lib/bash/home-manager.sh

```diff
@@ -83,3 +83,19 @@ function _iNote() {
     _i "$@"
     echo -n "${normalColor}"
 }
+
+# Runs the given command on live run, otherwise prints the command to standard
+# output.
+#
+# If given the command line option `--silence`, then the command's standard and
+# error output is sent to `/dev/null` on a live run.
+function dryRun() {
```

Comment:
more than `dryRun`, its a `runCommand` that might be a dryrun

Reply:
ie. not a fan of the name, bikeshedding a bit.

Reply:
Yes, I agree about the naming being poor, I was having difficulties finding a good naming scheme, though. An alternative I considered but figured maybe was too long was something like `dryEchoLiveRun`.

FWIW I'm thinking to add a `verboseEcho` as well (as a replacement of the `VERBOSE_ECHO` variable).

Reply:
Had a think and ended up with simply calling the function `run`. It seems more sensible to me.